app/main.py

```diff
@@ -1,3 +1,17 @@
 def cache(func):
-    # Write your code here
-    pass
+    cached_values = []
+
+    def inner(*args):
+
+        lst = [*args]
```

Comment:
You should not transform it into a list because it is already a tuple. Also, [*args] is a bad transformation tuple into a list, better use `list(args)`.